R/lazyframe-frame.R

```diff
@@ -264,14 +267,14 @@ lazyframe__sort <- function(
   })
 }
 
-#' Add columns to this LazyFrame
+#' Modify/append column(s) of a LazyFrame
 #'
-#' Added columns will replace existing columns with the same name.
+#' Add columns or modify existing ones with expressions. This is
+#' the equivalent of `dplyr::mutate()` as it keeps unmentioned columns (unlike
+#' `$select()`).
 #'
-#' Creating a new LazyFrame using this method does not create a new copy of
-#' existing data.
-#' @inherit pl__LazyFrame return
 #' @inheritParams lazyframe__select
+#' @inherit pl__LazyFrame return
```

Comment:
Why this change?
I think that inherit should be above inheritParams because it can inherit including params.

Reply:
I put the inherit first but I really don't think that matters since we didn't specify params in the inherit